projenrc/insert-task-step.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, Project, TaskStep } from 'projen';
+
+export interface InsertTaskStepProps {
+  readonly taskName: string;
+  readonly insertSteps: TaskStep[];
+  readonly beforeExec: string;
```

Comment:
Confusing name...to me this reads like "execute the provided task before this exec". When actually its "this is the exec right after the provided task"?